scrnaseq.Rmd

```diff
@@ -294,30 +312,93 @@ sc = subset(sc, features=features_keep)
 sc
 ```
 
-## Normalisation, feature selection and scaling {.tabset}
-Feature selection: For downstream analysis it is beneficial to focus on genes that exhibit high cell-to-cell variation, that is they are highly expressed in some cells and lowly expressed in others. 
+## Normalisation, cell cycle scoring, scaling and variable features {.tabset}
+In this section, we subsequently run a series of Seurat functions:  
+1. We start by running a __standard log normalisation__, where counts for each cell are divided by the total counts for that cell and multiplied by 10,000. This is then natural-log transformed.   
+2. To be able to compare normalised gene counts between genes, gene counts are further __scaled__ to have zero mean and unit variance (z-score). This way, genes are equally weighted for downstream analysis.  
+3. __Variable features__ are selected. For downstream analysis it is beneficial to focus on genes that exhibit high cell-to-cell variation, that is they are highly expressed in some cells and lowly expressed in others. 
+
+In addition, we run:  
+4. __SCTransform__, a new and more sophisticated normalisation method that replaces the previous functions (__normalisation, scaling and variable features__). Note that although results are kept for steps 1-3 ("RNA" assay), downstream analyses will default to resulting data of step 4 ("SCT" assay). 
```

Comment:
I have the feeling that we should abandon SCTransform for now? Or we could allow the user to choose a normalisation method?

Reply:
Why do you want to abandon SCTransform? 

Can we keep it in for this PR, and discuss this separately?

Reply:
Yes, when we have multiple samples, we have to think about it anyway, since it might be different techs